Libraries/D3D12RaytracingFallback/src/FallbackLayerUnitTests/fallbacklayerunittests.cpp

```diff
@@ -2769,6 +2770,78 @@ void AllocateUAVBuffer(ID3D12Device &d3d12device, UINT64 bufferSize, ID3D12Resou
             }
         }
 
+		TEST_METHOD(LoadTriangles)
+		{
+			ID3D12Device &d3d12Device = m_d3d12Context.GetDevice();
+			LoadPrimitivesPass loadPrimitivesPass(&d3d12Device, 1);
+
+			CComPtr<ID3D12GraphicsCommandList> pCommandList;
+			m_d3d12Context.GetGraphicsCommandList(&pCommandList);
+
+			std::vector<Triangle> triangles;
+			triangles.push_back({ -2.0f,  1.0f, -2.0f,  1.0f,  1.0f, -2.0f,  1.0f,  1.0f,  1.0f });
```

Comment:
This is nitpicky but consider taking all these and consolidating it to a for-loop where the triangle contents are initialized using the for-loop counter. This is nice because you'll just have a single const that will control the amount of triangles (so if later on we want to turn this into a stress test, you just update a single number rather than copy-paste a million times). Note that the triangle vertices don't have to actually form a meaningful triangle (since they're not used for rendering) so initialization should be straightforward